README.md

```diff
@@ -4,18 +4,58 @@ SourcePauseTool
 A plugin for all your pausing needs.
 
 ## Usage
-Place the compiled `spt.dll` into your mod directory (for example: `hl2`, `portal`, `ep2`), launch the game and type `plugin_load spt` into the developer console. For older engines you will need to place `spt.dll` into the topmost `bin` directory (the one with a lot of different DLL files including `engine.dll`, *not* the one with `client.dll` and `server.dll`).
+*Note: Steam, by default, installs games to `C:\Program Files (x86)\Steam\steamapps\common` on 64-bit Windows, and `C:\Program Files\Steam\steamapps\common` on 32-bit Windows.*
+
+1. Place the corresponding `spt*.dll` in the mod directory, *for example*:
```

Comment:
This point is overcrowded. I think it'd be better to separate the "which SPT for which game/mod" and the "which folder to place it in".